packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L1StandardBridge.sol

```diff
@@ -69,7 +70,8 @@ contract OVM_L1StandardBridge is iOVM_L1StandardBridge, OVM_CrossDomainEnabled {
      * Depositing *
      **************/
 
-    /// @dev Modifier requiring sender to be EOA.  This check could be bypassed by a malicious contract via initcode, but it takes care of the user error we want to avoid.
+    /// @dev Modifier requiring sender to be EOA.  This check could be bypassed by a malicious
```

Comment:
I would use the /** */ syntax here just so it's consistent with the rest of the file